openmicroscopy/openmicroscopy
# Reset password

Do not allow to change LDAP password

To test 
- Use the first user listed under https://www.openmicroscopy.org/site/team/Password
- Check that the LDAP info are displayed and you cannot change the password
- Log out
- log in as an admin
- Go to the Admin tab
- Expand Test Group and select the user used previously
- Check that the LDAP info are displayed. Check that you can neither modify the password  nor the login name.
